<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>-70</x>
      <y>20</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="verticalGroupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>142</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox_show_mesh">
       <property name="text">
        <string>Show mesh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_show_texture">
       <property name="text">
        <string>Show texture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>add Texture</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_show_illumination">
       <property name="text">
        <string>Show illlumination</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>251</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;     &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Models list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>251</width>
      <height>0</height>
     </size>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;     &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Camera list&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>251</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Use</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>21</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QMenuBar" name="menubar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>24</height>
     </rect>
    </property>
    <widget class="QMenu" name="fileMenu">
     <property name="title">
      <string>File</string>
     </property>
     <addaction name="actionLoad_Model"/>
     <addaction name="actionLoad_Texture"/>
     <addaction name="actionChose_Color"/>
     <addaction name="actionSave_Model"/>
     <addaction name="actionTriangulation"/>
     <addaction name="actionRotate_Scale_Translate"/>
     <addaction name="actiondsad"/>
    </widget>
    <widget class="QMenu" name="moveMenu">
     <property name="title">
      <string>Move</string>
     </property>
     <addaction name="actionUp"/>
     <addaction name="actionDown"/>
     <addaction name="actionLeft"/>
     <addaction name="actionRight"/>
     <addaction name="actionBack"/>
     <addaction name="actionForward"/>
    </widget>
    <addaction name="fileMenu"/>
    <addaction name="moveMenu"/>
   </widget>
   <widget class="QStatusBar" name="statusbar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>811</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal mode</string>
    </property>
   </widget>
   <action name="actionTriangulation">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Triangulation</string>
    </property>
    <property name="toolTip">
     <string>triangulation</string>
    </property>
   </action>
   <action name="actionLoad_Model">
    <property name="text">
     <string>Load Model</string>
    </property>
   </action>
   <action name="actionLoad_Texture">
    <property name="text">
     <string>Load Texture</string>
    </property>
   </action>
   <action name="actionChose_Color">
    <property name="text">
     <string>Chose Color</string>
    </property>
   </action>
   <action name="actionSave_Model">
    <property name="text">
     <string>Save Model</string>
    </property>
   </action>
   <action name="actionUp">
    <property name="text">
     <string>Up</string>
    </property>
    <property name="shortcut">
     <string>Up</string>
    </property>
   </action>
   <action name="actionDown">
    <property name="text">
     <string>Down</string>
    </property>
    <property name="shortcut">
     <string>Down</string>
    </property>
   </action>
   <action name="actionLeft">
    <property name="text">
     <string>Left</string>
    </property>
    <property name="shortcut">
     <string>Left</string>
    </property>
   </action>
   <action name="actionRight">
    <property name="text">
     <string>Right</string>
    </property>
    <property name="shortcut">
     <string>Right</string>
    </property>
   </action>
   <action name="actionBack">
    <property name="text">
     <string>Back</string>
    </property>
    <property name="shortcut">
     <string>S</string>
    </property>
   </action>
   <action name="actionForward">
    <property name="text">
     <string>Forward</string>
    </property>
    <property name="shortcut">
     <string>W</string>
    </property>
   </action>
   <action name="actionRotate_Scale_Translate">
    <property name="text">
     <string>Rotate_Scale_Translate</string>
    </property>
   </action>
   <action name="actiondsad">
    <property name="text">
     <string>dsad</string>
    </property>
   </action>
  </widget>
  <action name="actionRotateMesh">
   <property name="text">
    <string>TranslateMesh</string>
   </property>
  </action>
  <action name="actionTranslateMesh">
   <property name="text">
    <string>TranslateMesh</string>
   </property>
  </action>
  <action name="actionTranslateMesh_2">
   <property name="text">
    <string>TranslateMesh</string>
   </property>
  </action>
  <action name="actionTranslate_Mesh">
   <property name="text">
    <string>Translate_Mesh</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
